frontend/src/App.tsx

```diff
@@ -58,7 +58,7 @@ const App = () => {
                   {keycloak?.authenticated ? <AppNavBar /> : null}
                   <Col style={{ padding: 0 }}>
                     <Route path="/login" component={Login}></Route>
-                    <PrivateRoute path="/accessdenied" component={AccessDenied}></PrivateRoute>
+                    <Route path="/forbidden" component={AccessDenied}></Route>
```

Comment:
I prefer single words for URL path segments

Reply:
Anyone can be forbidden if they don't have permission.  Additionally if it becomes a PrivateRoute it causes an infinite loop.